rma_repair/models/repair.py

```diff
@@ -10,6 +10,7 @@ class RepairOrder(models.Model):
     rma_line_id = fields.Many2one(
         comodel_name="rma.order.line", string="RMA", ondelete="restrict"
     )
-    under_warranty = fields.Boolean(
+    rma_under_warranty = fields.Boolean(
+        string="Under Warranty",
         related="rma_line_id.under_warranty",
```

Comment:
This was making the repair one related to the linked RMA (at least that's the intention, I have not functionally check). In case, we don't want to disconnect them, you should fix only the view and keep only one.

If what I describe above (have both fields related) does not work as one would expect, a refactor with propor compute methods might be needed.